Python/getargs.c

```diff
@@ -1934,7 +1934,8 @@ new_kwtuple(const char * const *keywords, int total, int pos)
             Py_DECREF(kwtuple);
             return NULL;
         }
-        PyUnicode_InternInPlace(&str);
+        PyInterpreterState *interp = _PyInterpreterState_GET();
+        _PyUnicode_InternMortal(interp, &str);
```

Comment:
This string is actually immortal in sync with the life of main interpreter by 81865002aee8eaaeb3c7e402f86183afa6de77bf?